Simplified/LibraryFinder/UI/NYPLLibraryFinderViewController.swift

```diff
@@ -48,7 +49,6 @@ class NYPLLibraryFinderViewController: UICollectionViewController, UICollectionV
   }
 
   override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-    super.collectionView(collectionView, didSelectItemAt: indexPath)
```

Comment:
shouldn’t this stay? just wondering if there’s a reason i’m missing … even if super class impl. does nothing, it’s good to future proof it by calling super….

Reply:
This line is causing a `NSInvalidArgumentException` crash for some reason, even if this is only line being called in this method. Any thoughts?

Reply:
that's pretty weird, I'm guessing the index path somehow became invalid? even that doesn't make a lot of sense, but the exception hints at something to do with the arguments. is it at all possible that the collection view was dealloced and recreated? or some other thread is modifying the number of items to be displayed?

Reply:
I checked everything you mentioned above and none of them seems to be the case.
Could it be because `collectionView(_:didSelectItemAt:))` is an optional implementation for `UICollectionViewController`?
ref: https://stackoverflow.com/a/11403413

Reply:
I can add an if-statement `UICollectionViewController.responds(to: #selector(collectionView(_:didSelectItemAt:)))` before calling the `super.collectionView(_:didSelectItemAt:))`. What do you think?

Reply:
that's an interesting thought.... it's weird that the method though requires `override`... but if you're saying that with the check it doesn't crash that's probably it. I think though you wanna use `instancesRespond(to:)`?

Reply:
You are right, updated it with `instancesRespond(to:)`